{{cookiecutter.project_name}}/CITATION.cff

```diff
@@ -4,14 +4,21 @@ cff-version: "1.1.0"
 title: "{{ cookiecutter.project_name }}"
 authors:
   -
-    family-names: "{{ cookiecutter.full_name }}"
-    given-names:
+    family-names: {{ cookiecutter.full_name.split(' ')[-1] }}
+    given-names: {{ cookiecutter.full_name.split(' ')[0] }}
     orcid: "https://orcid.org/0000-0000-0000-0000"
 date-released: 20??-MM-DD
 doi: <insert your DOI here>
 version: "{{ cookiecutter.version }}"
 repository-code: "{{ cookiecutter.repository }}"
-identifiers:
 keywords:
+  - keyword1
+  - keyword2
 message: "If you use this software, please cite it using these metadata."
-license: "{{ cookiecutter.license }}"
+{{ { "Apache Software License 2.0": "license: Apache-2.0",
+    "MIT license": "license: MIT",
+    "BSD license": "license: BSD-3-Clause",
+    "ISC license": "license: ISC",
+    "GNU General Public License v3 or later": "license: GPL-3.0-or-later",
+    "Not open source": ""
+}[cookiecutter.license] }}
```

Comment:
This dictionary switch was needed because the license enum in CFF is not the same enum as that of the PyPI metadata